gattrib/src/x_window.c

```diff
@@ -115,7 +115,7 @@ x_window_init()
   /* This basically sets up the overhead for the sheets, as I understand
    * it.  The memory for the actual sheet cells is allocated later,
    * when gtk_sheet_new is invoked, I think.  */
-  sheets = g_malloc0(NUM_SHEETS * sizeof(GtkWidget *));
+  sheets = (GtkSheet**) g_malloc0 (NUM_SHEETS * sizeof (GtkSheet*));
```

Comment:
I tend to use the [`g_new0()` macro](https://developer.gnome.org/glib/2.50/glib-Memory-Allocation.html#g-try-new0) in situations like this. It does the required cast, so you don't have to type out the typename twice:

```c
sheets = g_new0(GtkSheet*, NUM_SHEETS);
```